projects/dotcms-field-elements/src/utils/props/validators/props.ts

```diff
@@ -28,6 +28,16 @@ export function dateValidator<T>(propInfo: PropValidationInfo<T>): void {
     }
 }
 
+export function dateRangeValidator<T>(propInfo: PropValidationInfo<T>): void {
```

Comment:
doc?

Really this function not validate a Range date, I mean when I read 'dateRangeValidator' I think that tis one validate a Max and Min date but really this validate date format for two date, I don't now if the name is the right one here